### PR TITLE
Improve snippets test coverage from tutorial to wrapper

### PR DESCRIPTION
**Pull request by 6hundreds**
Part of https://github.com/gradle/gradle/issues/21027

### CODE REVIEW

File: subprojects/docs/src/snippets/tutorial/antLoadfileWithMethod/tests/antLoadfileWithMethod.sample.conf [hunk 1]
```diff
@@ -2,6 +2,6 @@
 # gradle --quiet loadfile
 # end::cli[]
 executable: gradle
-args: loadfile
+args: checksum
```
Comment (6hundreds) [inline subprojects/docs/src/snippets/tutorial/antLoadfileWithMethod/tests/antLoadfileWithMethod.sample.conf:5]: Rethinking this

File: subprojects/docs/src/snippets/tutorial/manifest/tests/manifest.sample.conf [hunk 2]
```diff
@@ -1,2 +1,2 @@
 executable: gradle
-args: tasks
+args: fooJar barJar
```
Comment (mlopatkin) [inline subprojects/docs/src/snippets/tutorial/manifest/tests/manifest.sample.conf:2]: Shouldn't we also call `jar` task here as well?

**Review by mlopatkin — APPROVED** [review]
LGTM with a small suggestion

**Comment by bamboo** [comment]
@bot-gradle test this

File: subprojects/docs/src/snippets/tutorial/manifest/tests/manifest.sample.conf (hunk 2 above)
Comment (6hundreds) [reply subprojects/docs/src/snippets/tutorial/manifest/tests/manifest.sample.conf]: Fixed

**Comment by 6hundreds** [comment]
@bot-gradle test and merge

**Pull request merged**